src/ioc/index.ts

```diff
@@ -1,12 +1,15 @@
 import IoCContainer from 'ioc-lite';
 
-// import { Logger } from '../services/logger';
-// import { HTTP } from '../services/http';
-// import { Users } from '../services/users';
+import { Logger } from '../services/logger';
+import { HTTP } from '../services/http';
+import { Users } from '../services/users';
 
-export const createIoCContainer = () =>  {
-  const ioc = new IoCContainer();
-  // you can register some resources right now below...
+export const createIoCContainer = (): IoCContainer<any> => {
+  const ioc = new IoCContainer<any>();
```

Comment:
Please don't use any. 
We have to use generics to control which resources can be registered.

For example: if a developer registers HTTP service as a logger, TS will show an error: Argument of type 'typeof HTTP' is not assignable to parameter of type 'typeof Logger'.